### PR TITLE
Assignments for Week-5

### PR DESCRIPTION
**Pull request by Salmanz18**
Added Assignments for Week-5

### CODE REVIEW

File: Week-5/Exercise-5.4/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```
Comment (dev-ashishk) [inline Week-5/Exercise-5.4/.gitignore:2]: this file should be in the root of the project.

File: Week-5/Exercise-5.4/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (dev-ashishk) [inline Week-5/Exercise-5.4/package.json:1]: this file should be in the root of the project.

**Review by dev-ashishk — CHANGES_REQUESTED** [review]

**Comment by dev-ashishk** [comment]
@Salmanz18 why i am seeing week-4 changes in the pull requests ?

File: Week-5/Exercise-5.4/.gitignore (hunk 1 above)
Comment (Salmanz18) [reply Week-5/Exercise-5.4/.gitignore:2]: Actually, I used jest library for only a specific assignment so these files are in that assignments folder only.

File: Week-5/Exercise-5.4/package.json (hunk 2 above)
Comment (Salmanz18) [reply Week-5/Exercise-5.4/package.json:1]: Actually, I used jest library for only a specific assignment so these files are in that assignments folder only.

**Comment by Salmanz18** [comment]
I'm also not aware why that happened, It automatically came once I committed Week-5 changes though I used different branches for both the pull requests.

**Pull request merged**